### PR TITLE
Read ToDo Entry Component Feature

### PR DESCRIPTION
**Pull request by ksrscharan**
## Description
- Read Todo Component Added.

## Database schema changes
NA

### Automated test cases added
NA

### Manual test cases run
NA

### CODE REVIEW

**Comment by ksrscharan** [comment]
Screenshot 2022-06-27 at 1 35 00 PM
Screenshot 2022-06-27 at 1 36 39 PM

File: src/components/AddTodo.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from "react";
+import { ACTION } from "../context/Actions";
```
Comment (sourabh-4097) [inline src/components/AddTodo.tsx:3]: packages and local imports should be seperated by line from each other

Comment (sourabh-4097) [reply src/components/AddTodo.tsx:3]: example - 
import React, { useContext } from "react";

import { ACTION } from "../context/Actions";
import { GlobalContext } from "../context/GlobalState";

File: src/components/ReadTodo.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React, { useContext } from "react";
+import { ACTION } from "../context/Actions";
```
Comment (sourabh-4097) [inline src/components/ReadTodo.tsx:3]: follow above comment

**Pull request merged**